lib/resque/scheduler/env.rb

```diff
@@ -56,32 +56,22 @@ def setup_pid_file
 
       def setup_scheduler_configuration
         Resque::Scheduler.configure do |c|
-          if options.key?(:app_name)
```

Comment:
here we're explicitly using a config key IFF it is present (via `Hash#key?`), which is intentional because it allows us to explicitly use nil to unset a configuration variable; your changes make it so we cannot unset these config variables.